### PR TITLE
Small code cleanup and clarifications

### PR DESCRIPTION
**Pull request by klaren**

### CODE REVIEW

File: community/logging/src/main/java/org/neo4j/logging/async/AsyncLogEvent.java [hunk 1]
```diff
@@ -153,32 +152,20 @@ public String toString()
...
-        protected DateFormat initialValue()
-        {
-            SimpleDateFormat format = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.SSSZ" );
-            format.setTimeZone( TimeZone.getTimeZone( "UTC" ) );
-            return format;
-        }
+        return DATE_FORMAT_THREAD_LOCAL.get().format( new Date( timestamp ) );
     }
+
+    private static final ThreadLocal<DateFormat> DATE_FORMAT_THREAD_LOCAL = ThreadLocal.withInitial( () -> {
```
Comment (chrisvest) [inline community/logging/src/main/java/org/neo4j/logging/async/AsyncLogEvent.java:165]: This is very similar to the `org.neo4j.helpers.Format.date` family of methods.

Comment (klaren) [reply community/logging/src/main/java/org/neo4j/logging/async/AsyncLogEvent.java:165]: Sadly that resides in kernel and is not accessible in the logging module, and it's public so can't move it... :(

File: enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/modeswitch/HighAvailabilityModeSwitcher.java [hunk 2]
```diff
@@ -369,7 +369,7 @@ public void run()
                 }
                 catch ( HighAvailabilityStoreFailureException e )
                 {
-                    userLog.error( "UNABLE TO START UP AS SLAVE: %s", e.getMessage() );
+                    userLog.error( "UNABLE HEADER_TO START UP AS SLAVE: %s", e.getMessage() );
```
Comment (MishaDemianenko) [inline enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/modeswitch/HighAvailabilityModeSwitcher.java]: nope

File: community/kernel/src/main/java/org/neo4j/kernel/configuration/HttpConnector.java [hunk 3]
```diff
@@ -95,8 +95,8 @@ public Encryption encryptionLevel()
     {
         NONE( "http", 7474 ), TLS( "https", 7473 );
 
-        final String uriScheme;
-        final int defaultPort;
+        public final String uriScheme;
```
Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/configuration/HttpConnector.java:98]: not sure why those fields are now public

File: community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/NodeSchemaMatcher.java [hunk 4]
```diff
@@ -35,11 +35,9 @@
  */
 public class NodeSchemaMatcher
 {
-    private final Read read;
 
     public NodeSchemaMatcher( Read read )
```
Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/NodeSchemaMatcher.java]: if this class does not have any state now, should we have a possibility to create one? or static helper method should be enough?

File: community/kernel/src/main/java/org/neo4j/kernel/impl/util/NoneStrictMath.java [hunk 5]
```diff
@@ -26,7 +26,7 @@
  */
 public class NoneStrictMath
 {
-    public static double EPSILON = 1.0E-8;
+    public static final double EPSILON = 1.0E-8;
```
Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/util/NoneStrictMath.java]: 🙈 🙉 🙊

File: community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java [hunk 6]
```diff
@@ -130,7 +130,7 @@ void visit( ResourceType resourceType, long resourceId, String description, long
         /** For slave transactions, this tracks an identifier for the lock session running on the master */
         int getLockSessionId();
 
-        Stream<? extends ActiveLock> activeLocks();
+        Stream<ActiveLock> activeLocks();
```
Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java]: please restore previous `<? extends` thingy

**Review by MishaDemianenko — CHANGES_REQUESTED** [review]
Couple of small comments

File: enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/modeswitch/HighAvailabilityModeSwitcher.java (hunk 2 above)
Comment (klaren) [reply enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/modeswitch/HighAvailabilityModeSwitcher.java]: Did not anticipate shouting, reverting...

File: community/kernel/src/main/java/org/neo4j/kernel/impl/util/NoneStrictMath.java (hunk 5 above)
Comment (klaren) [reply community/kernel/src/main/java/org/neo4j/kernel/impl/util/NoneStrictMath.java]: :sob:

**Review by MishaDemianenko — APPROVED** [review]

**Pull request merged**